runtime/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ add_custom_command(OUTPUT copy_docfiles
     ${PROJECT_SOURCE_DIR}/runtime/doc ${GENERATED_RUNTIME_DIR}/doc
 )
 
-add_custom_command(OUTPUT ${GENERATED_HELP_TAGS}
+add_custom_target(helptags
```

Comment:
The tags file was not regenerating, so older helptags were being installed after a rebuild. In cmake, a "target" is always built, so this change ensures updated helptags are installed.